composer.json

```diff
@@ -24,7 +26,18 @@
         "phpstan-82": "phpstan/vendor/bin/phpstan -cphpstan/php82.neon analyse",
         "phpstan-83": "phpstan/vendor/bin/phpstan -cphpstan/php83.neon analyse",
         "tests-unit": "./vendor/bin/phpunit tests",
-        "tests-coverage": "export XDEBUG_MODE=coverage && php -d memory_limit=200M ./vendor/bin/phpunit tests --coverage-html ./tests-coverage"
+        "tests-coverage": "export XDEBUG_MODE=coverage && php -d memory_limit=200M ./vendor/bin/phpunit tests --coverage-html ./tests-coverage",
+        "check:all": [
+            "@check:phpcs",
+            "@tests-unit",
```

Comment:
podle mě by se měl spustit phpstan ještě před testy. Testy budou časem bobnat a budeš čekat třeba X minut "zbytečně", když ti to pak spadne na phpstanu a budeš se vrtat ve finále stejně v kodu a potažmo v testech